chocolatey/choco
# (#3218) Update Tab Expansion to use Test-Path

## Description Of Changes

Update Tab Expansion to use Test Path to check for a license file.

## Motivation and Context

The method we were using to clear the error variable wasn't clearing the variable.

## Testing

1. Check the contents of `$error` (and/or clear it with `$error.Clear()`).
2. Import ChocolateyProfile (from this repository you could do: `Import-Module src/chocolatey.resources/helpers/ChocolateyProfile.psm1 -Force`).
3. Check the contents of `$error` to ensure it hasn't logged any errors.
4. Add Chocolatey License file to system
5. Attempt to Tab-complete License commands (like Download)

### Operating Systems Testing

Server 2022

## Change Types Made


## Change Checklist


## Related Issue

Fixes #3218
